person.rb

```diff
@@ -23,5 +24,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Same here, This method suppose to add a new rental and the person itself should be sent as a parameter achieving the 'has-many' association. So please fix that.